src/babashka/fs.cljc

```diff
@@ -342,8 +343,9 @@
                                (or (str/includes? pattern "**")
                                    (str/includes? pattern file-separator)
                                    (when win?
-                                     (str/includes? pattern "/"))))]
-     (match root (str "glob:" pattern) (assoc opts :recursive recursive)))))
+                                     (str/includes? pattern "/"))))
+         hidden    (:hidden opts (when-not win? (str/starts-with? pattern ".")))]
```

Comment:
I'd say let's also do this for Windows, unless the hidden option is explicitly set?

Reply:
Yeah that makes sense I think.

I didn't do it as what I understood from glob's docstring is that being 'dotted' and 'hidden' are orthogonal things on Windows.  
So `.git*` already finds `.gitignore` and it can't be concluded from a dot-pattern that hidden files should be considered - but in the end we only add _dotfiles_ that are hidden to the haystack.

Reply:
Added, re-ran API generation and added some formatting to docstrings.